File: 4week/kiseok/javaScript/약수의개수와덧셈.js
function findNum(n){
    let cnt=0;
    for (let i=0; i<=n ; i++){
        if (n%i==0) cnt+=1;
    }
    return cnt;
}

function solution(left, right) {
    var answer = 0;
    for (let i = left; i<=right; i++)
        if(findNum(i)%2==0) answer+=i
        else answer-=i
    return answer;
}

File: 4week/kiseok/python/약수의개수와덧셈.py
# https://school.programmers.co.kr/learn/courses/30/lessons/77884

def find_num(n):
    cnt=0
    for i in range(1,n+1):
        if n%i==0:
            cnt+=1
    # print(n,cnt)
    return cnt

def solution(left, right):
    answer = 0
    for i in range(left,right+1):
        if find_num(i)%2==0:
            answer+=i
        else:
            answer-=i
    return answer